variants/STM32F7xx/F745Z(E-G)T_F746Z(E-G)(T-Y)_F750Z8T_F756ZG(T-Y)/variant_ETHERCAT_DUINO.cpp

```diff
@@ -0,0 +1,152 @@
+ * the "License"; You may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at:
+ *                        opensource.org/licenses/BSD-3-Clause
+ *
+ *******************************************************************************
+ */
+#if defined(ARDUINO_ETHERCAT_DUINO)
+#include "pins_arduino.h"
+
+// Digital PinName array
+const PinName digitalPin[] = {
+      PC_11  // D0    
+    , PC_10  // D1    
+    , PB_14  // D2    
+    , PD_12  // D3    
+    , PD_13  // D4    
+    , PD_14  // D5    
+    , PD_15  // D6    
+    , PC_6   // D7  
+    , PC_7   // D8  
+    , PC_8   // D9  
+    , PC_9   // D10 
+    , PA_8   // D11 
+    , PA_9   // D12 
+    , PA_15  // D13   
+    , PD_5   // D14 
+    , PD_6   // D15 
+    , PB_10  // D16   
+    , PB_11  // D17   
+    , PE_8   // D18 
+    , PE_7   // D19 
+    , PF_15  // D20   
+    , PF_14  // D21   
+    , PG_6   // D22 
+    , PG_7   // D23 
+    , PG_4   // D24 
+    , PG_5   // D25 
+    , PG_2   // D26 
+    , PG_3   // D27 
+    , PE_11  // D28   
+    , PE_15  // D29   
+    , PE_9   // D30 
+    , PE_10  // D31   
+    , PG_0   // D32 
+    , PG_1   // D33 
+    , PF_13  // D34   
+    , PF_12  // D35   
+    , PF_11  // D36   
+    , PG_9   // D37 
+    , PG_10  // D38   
+    , PG_11  // D39   
+    , PG_12  // D40   
+    , PG_13  // D41   
+    , PG_14  // D42   
+    , PF_10  // D43   
+    , PF_4   // D44 
+    , PF_3   // D45 
+    , PC_14  // D46   
+    , PC_15  // D47   
+    , PC_13  // D48   
+    , PE_6   // D49 
+    , PE_5   // D50 
+    , PE_4   // D51 
+    , PE_3   // D52 
+    , PE_2   // D53 
+};
+
+// Analog (Ax) pin number array
+const uint32_t analogInputPin[] = {
+      32 // A0 
+    , 33 // A1 
+    , 34 // A2 
+    , 35 // A3 
+    , 0  // A4 
+    , 1  // A5 
+    , 2  // A6 
+    , 3  // A7 
+    , 6  // A8 
+    , 7  // A9 
+    , 36 // A10
+    , 37 // A11
+};
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+  /**
+  * @brief  System Clock Configuration
+  * @param  None
+  * @retval None
+  */
+  WEAK void SystemClock_Config(void)
+  {
+    RCC_OscInitTypeDef RCC_OscInitStruct = {0};
+    RCC_ClkInitTypeDef RCC_ClkInitStruct = {0};
+    RCC_PeriphCLKInitTypeDef PeriphClkInitStruct = {0};
+
+    /** Configure the main internal regulator output voltage 
+  */
+    __HAL_RCC_PWR_CLK_ENABLE();
+    __HAL_PWR_VOLTAGESCALING_CONFIG(PWR_REGULATOR_VOLTAGE_SCALE1);
+    /** Initializes the CPU, AHB and APB busses clocks 
+  */
+    RCC_OscInitStruct.OscillatorType = RCC_OSCILLATORTYPE_HSE;
+    RCC_OscInitStruct.HSEState = RCC_HSE_ON;
+    RCC_OscInitStruct.PLL.PLLState = RCC_PLL_ON;
+    RCC_OscInitStruct.PLL.PLLSource = RCC_PLLSOURCE_HSE;
+    RCC_OscInitStruct.PLL.PLLM = 4;
+    RCC_OscInitStruct.PLL.PLLN = 216;
+    RCC_OscInitStruct.PLL.PLLP = RCC_PLLP_DIV2;
+    RCC_OscInitStruct.PLL.PLLQ = 9;
+    if (HAL_RCC_OscConfig(&RCC_OscInitStruct) != HAL_OK)
+    {
+    }
```

Comment:
Fixiing Astyle issue and add the error handler call.

```suggestion
    if (HAL_RCC_OscConfig(&RCC_OscInitStruct) != HAL_OK)  {
      Error_Handler();
    }
```